homeassistant/components/vicare/__init__.py

```diff
@@ -36,7 +37,7 @@
 def setup(hass, config):
     """Create the ViCare component."""
     conf = config[DOMAIN]
-    params = {"token_file": "/tmp/vicare_token.save"}
+    params = {"token_file": tempfile.gettempdir() + "/vicare_token.save"}
```

Comment:
I suggest we use `pathlib.Path` to construct the path.
```suggestion
    params = {"token_file": str(Path(tempfile.gettempdir()) / "vicare_token.save")}
```

Reply:
fine by me 
we just need to don't forget to import the pathlib